lib/helpers/utils.js

```diff
@@ -4624,9 +4625,9 @@ export function parsePixiLockFile(pixiLockFileName, path) {
         },
       },
       properties: [
-        { operatingSystem: packageData["subdir"] },
-        { build_number: packageData["build_number"] },
-        { build: packageData["build"] },
+        { name: "cdx:pixi:operating_system", value: packageData["subdir"] },
```

Comment:
Properties must be of the form name, value. Not sure how this escaped validation for so long.